test/release/examples/primers/chpldoc.chpl

```diff
@@ -36,21 +35,31 @@ proc commented(val: int): string {
 }
 
 /*
-  The method can be a stub and still output its comment
+  The function can be a stub and still output its comment
 */
 proc stub(val): bool {
-  /* Comments within the method body are ignored */
+  /* Comments within the function body are ignored */
 }
 
 // Single line comments are also ignored.  However, this does not prevent 
-// the display of the method itself.
+// the display of the function itself.
 proc uncommented() {
   /*
-    Longer comments within the method body are also ignored
+    Longer comments within the function body are also ignored
   */
 }
 
 /*
+  To prevent the display of a particular function, class, record, specified
+  module, or global variable, simply preface it with:
+  pragma "no doc"
+*/
+pragma "no doc"
+proc undocumented() {
+  // This function won't be in the final output.
+}
```

Comment:
It might be worth adding a comment stating that, once Chapel has support for marking symbols as private, that technique will be used to squash documentation instead of the pragma (maybe putting it in the README as well would calm my nerves about documenting a pragma as well).